### DISCUSSION TITLE
Remove Unlicense from list of suggested Licenses on creating a repo

### DISCUSSION THREAD

**Discussion by stokito** [community/community#53778] [Repositories]
### Select Topic Area

Bug

### Body

When creating a new repo you can select a license from list. This is a fair list of popular licenses except of the last option which is Unlicense.

People may think that this is something similar to a public domain and the most permissive license but it has many problems.

> Unlicense, which appears to borrow one paragraph/idea from CC0 and inappropriately merges it into the MIT license, is unvetted language and should not be used for any real software products.

https://github.com/gogs/gogs/issues/4687#issuecomment-462783802

Here is a longer discussion
* https://softwareengineering.stackexchange.com/questions/147111/what-is-wrong-with-the-unlicense

Please remove the controversial license from the list of suggestions. This makes people think that they are safe legally but they don't. It's misleading and may be considered as an attack on open source projects.

Ideally Github should let authors know about usage of bad licenses. For example WTFPL, Beerware and even some Public Domain like CC0 should be avoided. This is a kind of legal security vulnerability.

But also this increases fragmentation of licenses which leads to a weakened community and increases a burden: now I have to spend my time learning a new license to understand if it suits my needs. This is applied to many other licenses.For example I just saw a few Perl projects who use an Artistic license because the save license was used by Perl itself. But instead they may use a similar but more known MIT.Github may suggest the transition to authors.

See related discussions:
* https://github.com/awesome-selfhosted/awesome-selfhosted/issues/3753#issuecomment-1523014745
* https://github.com/github/choosealicense.com/issues/805#issuecomment-797730371

**Comment by mg0x7BE** [comment]
The mere fact that something may be controversial does not mean it cannot be useful. This is my favorite license because it is understandable for an ordinary person, not just a lawyer. Even if it might cause issues in strange jurisdictions, I still prefer to use it because it is simple and clear, unlike legal jargon.
